packages/core/src/controllers/relayer.ts

```diff
@@ -58,6 +58,7 @@ export class Relayer extends IRelayer {
 
   private relayUrl: string;
   private projectId: string | undefined;
+  public transportExplicitlyClosed = false;
```

Comment:
Since this is public (does it need to be, i.e. are we expecting people to check this?), can we move it up to live with the other public props?

Reply:
It's a flag to differentiate why the connection might be down.
Checking `relayer.connected()` is not sufficient as it might be down due to several reasons such as
- socket died/invalid relay url/other network issue
- closed via `transportClose`

Reply:
Yeah makes sense :) 👍